templates/private/Tasks/EndTask/task.json

```diff
@@ -32,6 +32,13 @@
       "required": true,
       "helpMarkDown": "Comma separated list of target Octane workspaces"
     },
+    {
+      "name": "CucumberReportPath",
+      "type": "string",
+      "label": "Cucumber report destination path",
+      "required": true,
```

Comment:
It should be required: false. There might be clients that will never use cucumber. Make sure you support both cases, with and without cucumber tests